ios/RNSScreenStackHeaderConfig.mm

```diff
@@ -10,7 +18,13 @@
 #import <React/RCTShadowView.h>
 #import <React/RCTUIManager.h>
 #import <React/RCTUIManagerUtils.h>
+#import "RNSSearchBar.h"
+#endif
+#import "./utils/RNSUIBarButtonItem.h"
```

Comment:
Should this import look like that? Can't it be just `#import "RNSUIBarButtonItem.h"`

Reply:
[6818798](https://github.com/software-mansion/react-native-screens/pull/1417/commits/68187987c0a9dd767ad4df46d25bd205fc0781f1)